app/gui/view/graph-editor/src/builtin/visualization/native/text_visualization/grid_view_entry.rs

```diff
@@ -77,6 +77,7 @@ impl Entry {
         let mut style = "position: absolute; white-space: pre; pointer-events: auto;".to_string();
         write!(style, "left: {left}px; top: {top}px;").ok();
         write!(style, "width: {width}px; height: {height}px;").ok();
+        write!(style, "line-height: 1").ok();
```

Comment:
What is this? Looks magical. If we need it, it should have doc comment with explanation.

Reply:
I explained in the PR description, but I'll copy-paste it into a comment.

Reply:
Hmm, if you are referring to this explanation:

```
The line height of text grid entries was set to the default of 1.2. That's the default line height in browsers, which is great for multi-line text and elements whose height is greater than the line height. In this case, however, where the height and the font size are set to the same value, the default setting of 1.2 pushes the text below the allotted space.
```

Then I do not understand it. I was reading it several times and it's not clear to me. default of 1.2 is great for multi-line text readability, like in our case. We should not change it to 1.0. 

Anyway, you've written "In this case, however, where the height and the font size are set to the same value, the default setting of 1.2 pushes the text below the allotted space":
- Height of what is the same as the font size?
- What is "allotted space"?
- Why these things are "pushed below" this space?

Can we make it working with default line-height?

Reply:
> * Height of what is the same as the font size?

The height of the element this part of the code is defining: the grid view entry.

> * What is "allotted space"?

The vertical space that is available for the text in this element which is fixed by the height which is explicitly set in the line above.

> * Why these things are "pushed below" this space?

Because with a font size of 12px and a line height of 1.2, there's not a enough room in an element that's only 12px high. It needs something like `1.2 * 12px ~= 14px`.

> Can we make it working with default line-height?

Another option might be to make the entry size higher: not set it to any value and let the browser derive it from the font size and line height. Looking at the code, however, it looks like we always let the entry size be set externally, by the component that uses the text grid. That might be substantial change because the grid view is used in a bunch of places. Any other ideas?

Reply:
Wait, can't we just change this line:
https://github.com/enso-org/enso/blob/develop/lib/rust/ensogl/app/theme/hardcoded/src/lib.rs#L590 ?

It sets the height of the grid entry as far as I can see.

Reply:
@wdanilo: To make sure there's no confusion here, this PR doesn't change the distance between lines of text. The grid view entries are still fixed to the same height. This just changes where the text appears in relation to the grid view entry div.